src/components/notification/notification.stories.ts

```diff
@@ -191,9 +181,21 @@ export const SlottedTitle: StoryObj = {
   args: { ...basicArgs, 'title-content': undefined },
 };
 
+// We set the height of the div in Chromatic to avoid cropped snapshots
 const meta: Meta = {
   decorators: [
-    (story) => html`<div style="padding: 2rem;">${story()}</div>`,
+    (story, context) =>
+      html`<div
+        style="padding: 2rem;display: flex;gap: var(--sbb-spacing-fixed-4x);flex-direction: column;${isChromatic()
```

Comment:
Just a comment: I noticed that now the expansion animation is a bit laggy, is it because of the `flex` here?